Dockerfile

```diff
@@ -10,10 +10,15 @@ RUN apt-get update && \
     apt-get install python3-catkin-tools -y
 
 # Install ROS dependencies
-RUN apt-get install -y ros-noetic-rviz \
+RUN apt-get install -y --fix-missing \
```

Comment:
These dependencies take too much time to build. Also I got an error when running the docker file. may I suggest breaking them up in two instructions and remove the `--fix-missing` option.